tests/vankeken_subduction.prm

```diff
@@ -0,0 +1,14 @@
+
+include $ASPECT_SOURCE_DIR/cookbooks/vankeken_subduction/vankeken_corner_flow.prm
```

Comment:
The problem with the failing test is here. In this test, you are including the `.prm` file from the cookbooks directory, which contains the following lines:
```
# Load the prescribed velocity and pressure library
set Additional shared libraries            = ./plugin/libvankeken_subduction.so
```
But in the directory where the test is executed, there is no such shared library and the test consequently aborts. I'm surprised that the test works for you locally, if you run it with the commands
```
make setup_tests
ctest -R vankeken
```
in your build directory. 

This kind of test design doesn't work because it assumes that the shared library you want is located in a directory specified in the *cookbook*, not in the *test*. Are there other tests that reference `.prm` files from the cookbooks where we could look up how this is handled there?

Reply:
Put a .cc with the same name in the test directory and use an include statement of the source file in the cookbooks directory.